File: Java-Core/src/main/java/presto/parser/EParserFactory.java
package presto.parser;

import org.antlr.v4.runtime.ANTLRInputStream;

public class EParserFactory implements IParserFactory {

	@Override
	public ILexer newLexer() {
		return new EIndentingLexer(new ANTLRInputStream(""));
	}

	@Override
	public IParser newParser() {
		return new ECleverParser("");
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/parser/EParserFactory.js
var antlr4 = require("antlr4/index");
var EIndentingLexer = require("./EIndentingLexer").EIndentingLexer;
var ECleverParser = require("./ECleverParser").ECleverParser;

exports.EParserFactory = function() {
	
	this.newLexer = function(data) {
		return new EIndentingLexer(new antlr4.InputStream(data));
	};

	this.newParser = function(path, data) {
		return new ECleverParser(path, data);
	};

};
